include/dca/math/nfft/kernels_interface.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018 ETH Zurich
+// Copyright (C) 2018 UT-Battelle, LLC
+// All rights reserved.
+//
+// See LICENSE.txt for terms of usage.
+// See CITATION.txt for citation guidelines if you use this code for scientific publications.
+//
+// Author: Giovanni Balduzzi (gbalduzz@gitp.phys.ethz.ch)
+//
+// This file provides the interface to the kernels for the Dnfft1DGpu accumulator
+
+#ifndef DCA_MATH_NFFT_KERNELS_INTERFACE_HPP
+#define DCA_MATH_NFFT_KERNELS_INTERFACE_HPP
+#ifdef DCA_HAVE_CUDA
+
+#include <cuda.h>
+
+namespace dca {
+namespace math {
+namespace nfft {
+namespace details {
+// dca::math::nfft::details::
+
+struct ConfigElem {
+  int b, r;
```

Comment:
It is clearer to rename the variables: b --> band, r --> site / position.

Reply:
renamed in 1eb76429faaefa254286bfda76ff0bc2be6ed0f4